include/core/CImmutableRadixSet.h

```diff
@@ -1,146 +0,0 @@
-/*
```

Comment:
Would you please mention in the PR description why this file gets deleted? It is not obvious.

Reply:
This was to provide optimisation for `addRowDerivatives` but since computing splits is no longer on a hot path I simplified and just use `std::upper_bound`.